background.js

```diff
@@ -21,12 +21,19 @@ function listen(lang) {
   )
 }
 
+chrome.storage.sync.get(["language1", "language2"],
+function(items){
+    language1 = items.language1;
+    language2 = items.language2;
+});
+
 chrome.commands.onCommand.addListener(function(command) {
+//ตรวจภาษา
```

Comment:
You can move the lines 24-27 here so that we load the language settings every time, and move the line 28 to after line 37. It looks like this:

```js
chrome.commands.onCommand.addListener(function(command) {
  chrome.storage.sync.get(..., function(items) {
    if (...) {
      ...
    }
    ...
  })
})